azure-pipelines.yml

```diff
@@ -1,5 +1,5 @@
 trigger:
-  - master
+  - feature
```

Comment:
does this mean the tests only get triggered by commits to the feature branch? Is it possible to completely omit the `trigger` section?

Reply:
It does limit to feature, so this should be removed on merge with develop